CHANGELOG.md

```diff
@@ -2,6 +2,44 @@
 
 All notable changes to this project are documented in this file.
 
+## 0.23.0
+
+**Release date:** 2022-04-05
+
+This prerelease adds some breaking changes around the use and handling of kubeconfigs 
+files for remote reconciliations. Implements health checks for kubernetes jobs and 
+updates documentation.
+
+SOPS implementation was refactored to include various improvements and extended code
+coverage. A new `ParsedIdentities` utility was introduced, allowing for age identities
+to be imported once, and reused multiple times, optimising CPU and memory usage between
+decryption operations.
+
+**Breaking changes**:
+
+- Use of file-based KubeConfig options are now permanently disabled (e.g. 
+`TLSClientConfig.CAFile`, `TLSClientConfig.KeyFile`, `TLSClientConfig.CertFile`
+and `BearerTokenFile`). The drive behind the change was to discourage
+insecure practices of storing crypto material in disk, users should instead
+use the in-memory alternatives instead and follow best practices around
+encrypting and keeping such information protected.
```

Comment:
```suggestion
and `BearerTokenFile`). The drive behind the change was to discourage
insecure practices of mounting Kubernetes tokens inside the controller's container file system.
```

Reply:
Applied the changes at both KC and HC PRs. PTAL

Reply:
thanks!